srcpkgs/webhook/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'webhook'
+pkgname=webhook
+version=2.8.0
+revision=1
+build_style=go
+go_import_path="github.com/adnanh/webhook"
+short_desc="Lightweight incoming webhook server to run shell commands"
+maintainer="Abigail G <dev@kb6.ee>"
+license="MIT"
+homepage="https://github.com/adnanh/webhook"
+distfiles="https://github.com/adnanh/webhook/archive/${version}.tar.gz"
+checksum=c521558083f96bcefef16575a6f3f98ac79c0160fd0073be5e76d6645e068398
+conf_files="/etc/webhook/hooks.json /etc/sv/webhook/conf"
+system_accounts="_webhook"
+system_groups="_webhook"
+
+post_install() {
+	vlicense LICENSE
+	vsv webhook
+
+	vinstall ${FILESDIR}/hooks.json 644 etc/webhook
+	for f in docs/*.md; do vdoc $f; done
+	for f in *.example; do vsconf $f ${f%.example}; done
```

Comment:
We don't need the .tmpl files
```
%: xbps-query -f webhook
...
/usr/share/examples/webhook/hooks.json
/usr/share/examples/webhook/hooks.json.tmpl
/usr/share/examples/webhook/hooks.yaml
/usr/share/examples/webhook/hooks.yaml.tmpl
...
```

Reply:
the tmpl files are examples of using go templating for hooks